docs/source/programming_resources/tutorial_specific/blocks/managing_opmodes/managing-opmodes.rst

```diff
@@ -1,5 +1,5 @@
-Managing OpModes in Blocks
-==========================
+Managing OpModes in Blocks :bdg-warning:`Blocks`
+=================================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".